vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/codec/json/JsonObjectCodec.java

```diff
@@ -106,6 +149,14 @@ protected Object readObjectId(BsonReader reader, DecoderContext ctx) {
 
   @Override
   protected Object readDateTime(BsonReader reader, DecoderContext ctx) {
```

Comment:
Back to the starting point :) This is a limitation of the mongo parser :
> The MongoDB JSON parser currently does not support loading ISO-8601 strings representing dates prior to the Unix epoch

 Of course I can implement that, but I think it will be confusing for vert.x users. The java mongo driver takes this complexity for us. With this "feature", helpers would be mandatory (and what about other languages?). Are you really sure you want it? :)